src/uct/rocm/copy/rocm_copy_md.c

```diff
@@ -25,19 +27,24 @@ static ucs_config_field_t uct_rocm_copy_md_config_table[] = {
      ucs_offsetof(uct_rocm_copy_md_config_t, super),
      UCS_CONFIG_TYPE_TABLE(uct_md_config_table)},
 
+    {"RCACHE", "try", "Enable using memory registration cache",
+     ucs_offsetof(uct_rocm_copy_md_config_t, enable_rcache),
+     UCS_CONFIG_TYPE_TERNARY},
+
     {NULL}
 };
 
 static ucs_status_t uct_rocm_copy_md_query(uct_md_h md, uct_md_attr_t *md_attr)
 {
-    md_attr->cap.flags            = UCT_MD_FLAG_REG;
-    md_attr->cap.reg_mem_types    = UCS_BIT(UCS_MEMORY_TYPE_HOST);
+    md_attr->cap.flags            = UCT_MD_FLAG_REG | UCT_MD_FLAG_NEED_RKEY;
+    md_attr->cap.reg_mem_types    = UCS_BIT(UCS_MEMORY_TYPE_HOST) |
+                                    UCS_BIT(UCS_MEMORY_TYPE_ROCM);
```

Comment:
Adding registraion capability for UCS_MEMORY_TYPE_ROCM to rocm_copy md disables rocm_ipc (I believe since rocm_copy can do get/put zcopy). Is there a way to still use rocm_ipc for intra-node rndv transfers?